ledger/complete/wal/wal.go

```diff
@@ -93,14 +93,15 @@ func (w *LedgerWAL) Replay(
 }
 
 func (w *LedgerWAL) ReplayLogsOnly(
+	checkpointFn func(forestSequencing *flattener.FlattenedForest) error,
 	updateFn func(update *ledger.TrieUpdate) error,
 	deleteFn func(rootHash ledger.RootHash) error,
 ) error {
 	from, to, err := w.wal.Segments()
 	if err != nil {
 		return err
 	}
-	return w.replay(from, to, nil, updateFn, deleteFn, false)
+	return w.replay(from, to, checkpointFn, updateFn, deleteFn, false)
```

Comment:
when replaying the transactions do we really want to use checkpointer ?

Reply:
Yes, for possible `root.checkpoint`